commit bcb42ed0dd0f228cd56776ef945268ed511d70e7 (refs/changes/52/4452/1)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-06-28T18:27:13+00:00 (1 year, 3 months ago)
    
    joyent/node-bcrypt-pbkdf#4 Link to GitHub repository from npm page

diff --git a/CONTRIBUTING.md b/CONTRIBUTING.md
new file mode 100644
index 0000000..401d34e
--- /dev/null
+++ b/CONTRIBUTING.md
@@ -0,0 +1,13 @@
+# Contributing
+
+This repository uses [cr.joyent.us](https://cr.joyent.us) (Gerrit) for new
+changes. Anyone can submit changes. To get started, see the [cr.joyent.us user
+guide](https://github.com/joyent/joyent-gerrit/blob/master/docs/user/README.md).
+This repo does not use GitHub pull requests.
+
+See the [Joyent Engineering
+Guidelines](https://github.com/joyent/eng/blob/master/docs/index.md) for general
+best practices expected in this repository.
+
+If you're changing something non-trivial or user-facing, you may want to submit
+an issue first.
diff --git a/README.md b/README.md
index 1201809..ab4f335 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,5 @@
 Port of the OpenBSD `bcrypt_pbkdf` function to pure Javascript. `npm`-ified
-version of [Devi Mandiri's port]
-(https://github.com/devi/tmp/blob/master/js/bcrypt_pbkdf.js),
+version of [Devi Mandiri's port](https://github.com/devi/tmp/blob/master/js/bcrypt_pbkdf.js),
 with some minor performance improvements. The code is copied verbatim (and
 un-styled) from Devi's work.
 
diff --git a/package.json b/package.json
index 51e8141..e93a969 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,11 @@
 {
   "name": "bcrypt-pbkdf",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "Port of the OpenBSD bcrypt_pbkdf function to pure JS",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/joyent/node-bcrypt-pbkdf.git"
+  },
   "main": "index.js",
   "dependencies": {
     "tweetnacl": "^0.14.3"
